app/lib/pricer/pricestf/prices-tf-api.js

```diff
@@ -50,21 +50,12 @@ class PricesTfApi {
                 ...headers,
             });
         } catch (e) {
-            if (e && 401 === e['statusCode']) {
```

Comment:
you should have kept this in, we need to reauthenticate

Reply:
Ok added it back!